modules/p2/ding_interaction/ding_interaction.api.inc

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * @file
+ * This file contains the api functions.
+ *
+ * The hooks can be used to define and provide new interaction content panes,
+ * and to alter or implement new output.
+ */
+
+/**
+ * Hook which defines ding interaction content.
+ *
+ * @return array
+ *   The returned array should contain the following values.
+ *   - "title": The title to display when the interaction pane is display.
+ *   - "active": Status of this interaction content.
+ *     TRUE if this pane should be displayed, FALSE if not.
+ */
+function hook_ding_interaction_info() {
+  return array(
+    'title' => t('Title to display'),
+    'active' => TRUE,
```

Comment:
@martinb14: This question still stands.

Documentation is greatly appreciated but it is most useful when it is complete and up to date.